code/game/gamemodes/gang/gang.dm

```diff
@@ -0,0 +1,366 @@
+									icon_state_to_use = "wanted_1"
+								if(TWO_STARS_HIGHPOP to THREE_STARS_HIGHPOP-1)
+									icon_state_to_use = "wanted_2"
+								if(THREE_STARS_HIGHPOP to FOUR_STARS_HIGHPOP-1)
+									icon_state_to_use = "wanted_3"
+								if(FOUR_STARS_HIGHPOP to FIVE_STARS_HIGHPOP-1)
+									icon_state_to_use = "wanted_4"
+								if(FIVE_STARS_HIGHPOP to INFINITY)
+									icon_state_to_use = "wanted_5"
+						else
+							switch(GLOB.deaths_during_shift)
+								if(0 to TWO_STARS_LOW-1)
+									icon_state_to_use = "wanted_1"
+								if(TWO_STARS_LOW to THREE_STARS_LOW-1)
+									icon_state_to_use = "wanted_2"
+								if(THREE_STARS_LOW to FOUR_STARS_LOW-1)
+									icon_state_to_use = "wanted_3"
+								if(FOUR_STARS_LOW to FIVE_STARS_LOW-1)
+									icon_state_to_use = "wanted_4"
+								if(FIVE_STARS_LOW to INFINITY)
+									icon_state_to_use = "wanted_5"
+						if(cops_arrived)
+							icon_state_to_use += "_active"
+							lock_stars = TRUE
+						current_stars = icon_state_to_use
+						H.wanted_lvl.icon_state = icon_state_to_use
+
+	check_counter++
+	if(check_counter >= 5)
+		if (world.time > endtime && !fuckingdone)
+			fuckingdone = TRUE
+			send_in_the_fuzz()
+		check_counter = 0
+		SSticker.mode.check_win()
+
+		check_tagged_turfs()
+		check_gang_clothes()
+		check_rollin_with_crews()
+
+/datum/game_mode/gang/proc/send_in_the_fuzz()
+	var/team_size
+	var/cops_to_send
+	if(GLOB.joined_player_list.len > LOWPOP_FAMILIES_COUNT)
+		switch(GLOB.deaths_during_shift)
+			if(0 to TWO_STARS_HIGHPOP-1)
+				team_size = 8
+				cops_to_send = /datum/antagonist/ert/families/beatcop
+			if(TWO_STARS_HIGHPOP to THREE_STARS_HIGHPOP-1)
+				team_size = 9
+				cops_to_send = /datum/antagonist/ert/families/beatcop/armored
+			if(THREE_STARS_HIGHPOP to FOUR_STARS_HIGHPOP-1)
+				team_size = 10
+				cops_to_send = /datum/antagonist/ert/families/beatcop/swat
+			if(FOUR_STARS_HIGHPOP to FIVE_STARS_HIGHPOP-1)
+				team_size = 11
+				cops_to_send = /datum/antagonist/ert/families/beatcop/fbi
+			if(FIVE_STARS_HIGHPOP to INFINITY)
+				team_size = 12
+				cops_to_send = /datum/antagonist/ert/families/beatcop/military
+	else
+		switch(GLOB.deaths_during_shift)
+			if(0 to TWO_STARS_LOW-1)
+				team_size = 5
+				cops_to_send = /datum/antagonist/ert/families/beatcop
+			if(TWO_STARS_LOW to THREE_STARS_LOW-1)
+				team_size = 6
+				cops_to_send = /datum/antagonist/ert/families/beatcop/armored
+			if(THREE_STARS_LOW to FOUR_STARS_LOW-1)
+				team_size = 7
+				cops_to_send = /datum/antagonist/ert/families/beatcop/swat
+			if(FOUR_STARS_LOW to FIVE_STARS_LOW-1)
+				team_size = 8
+				cops_to_send = /datum/antagonist/ert/families/beatcop/fbi
+			if(FIVE_STARS_LOW to INFINITY)
+				team_size = 10
+				cops_to_send = /datum/antagonist/ert/families/beatcop/military
+
+	var/list/mob/dead/observer/candidates = pollGhostCandidates("Do you want to help clean up crime on this station?", "deathsquad", null)
+
+	if(candidates.len > 0)
```

Comment:
Remove the comparison. Just check for 0.

Reply:
👍